org.subsurface_divelog.Subsurface.json

```diff
@@ -0,0 +1,122 @@
+{
+	"app-id": "org.subsurface_divelog.Subsurface",
+	"runtime": "org.kde.Platform",
+	"runtime-version": "5.9",
+	"sdk": "org.kde.Sdk",
+	"command": "subsurface",
+	"rename-icon": "subsurface-icon",
+	"rename-appdata-file": "subsurface.appdata.xml",
+	"rename-desktop-file": "subsurface.desktop",
+	"finish-args":[
+		"--socket=wayland",
+		"--socket=x11",
+		"--share=network",
+		"--device=dri",
+		"--filesystem=home",
+		"--device=all",
+		"--env=QTWEBENGINE_DISABLE_SANDBOX=1"
+	],
+	"build-options": {
+		"cflags": "-O2",
+		"cxxflags": "-O2"
+	},
+	"cleanup": [
+		"/include",
+		"/lib/pkgconfig",
+		"/lib64/pkgconfig",
+		"/lib/cmake",
+		"/share/doc",
+		"/share/man",
+		"*.a",
+		"*.la",
+		"*.cmake"
+	],
+	"modules":[
+		"shared-modules/udev/udev-175.json",
+		{
+			"name":"libssh2",
+			"buildsystem":"cmake",
+			"config-opts":[
+				"-DCMAKE_INSTALL_LIBDIR=/app/lib",
+				"-DCMAKE_C_FLAGS=-fPIC"
+			],
+			"sources":[
+				{
+					"type":"git",
+					"url":"https://github.com/libssh2/libssh2.git",
+					"branch": "libssh2-1.8.0",
+					"commit": "72274af7f901315c1d47e43620c5bcb5f40c2b20"
+				}
+			]
+		},
+		{
+			"name":"grantlee",
+			"buildsystem":"cmake",
+			"sources":[
+				{
+					"type":"git",
+					"url":"https://github.com/steveire/grantlee.git",
+					"branch": "v5.1.0",
+					"commit": "8f66fa86a24775e245df560771f313fa59770096"
+				}
+			]
+		},
+		{
+			"name":"libzip",
+			"sources":[
+				{
+					"type":"git",
+					"url":"https://github.com/nih-at/libzip.git",
+					"branch": "rel-1-2-0",
+					"commit": "a23ac8a766c556827255111eb35ba928641efbc8"
+				},
+				{
+					"type":"script",
+					"commands":["autoreconf -vfi", "./configure --prefix=/app", "make"],
+					"dest-filename": "autogen.sh"
+				}
+			]
+		},
+		{
+			"name": "subsuface",
+			"buildsystem":"simple",
+			"sources":[
+				{
+					"type":"git",
+					"url": "https://github.com/Subsurface-divelog/subsurface.git",
```

Comment:
Needs to be a stable release.

Reply:
The latest release fails to build, so I'd just wait for the next release before creating a Flathub repo.

Reply:
Depending on the timeline it might be acceptable to build from git but it has to be a specific commit not HEAD.

Reply:
Done. I don't know when they'll make another release, but the last one already dates back a bit.
Could this be merged now, or are the ARM builds of KDE still missing?

Reply:
ARM builds are still missing.

Reply:
The ARM builds are fixed now: https://github.com/flatpak/flatpak/issues/947#issuecomment-327037930

While testing I noticed that the googlemaps QML plugin was still missing and trying to crete a new log entry freezes the window.

Reply:
The map works now, the freeze has been reported upstream and it's not caused by the packaging:

https://github.com/Subsurface-divelog/subsurface/issues/565
http://lists.subsurface-divelog.org/pipermail/subsurface/2017-September/029924.html